pilot/proxy/envoy/mixer.go

```diff
@@ -201,13 +201,13 @@ func mixerHTTPRouteConfig(role proxy.Node, instances []*model.ServiceInstance, o
 					},
 				},
 			},
-			DisableCheckCalls:  outboundRoute,
-			DisableReportCalls: outboundRoute,
+			DisableCheckCalls:  outboundRoute || mesh.DisablePolicyChecks,
+			DisableReportCalls: outboundRoute || role.Type == proxy.Ingress,
```

Comment:
We may need to send Report for Egress too.

Reply:
I don't see an easy way to do this with the current code. It should be easier to do once we switch over to per-route config.

Reply:
OK. then.